sensei/ADIOS2Schema.cxx

```diff
@@ -422,7 +422,7 @@ int InputStream::Initialize(MPI_Comm comm)
   sensei::TimeEvent<128> mark("senseiADIOS2::InputStream::Initialize");
 
   // initialize adios2
-#if ADIOS2_VERSION_MAJOR > 2 || (ADIOS2_VERSION_MINOR == 2 && ADIOS2_VERSION_MINOR >= 9)
+#if ADIOS2_VERSION_MAJOR > 2 || (ADIOS2_VERSION_MAJOR == 2 && ADIOS2_VERSION_MINOR >= 9)
```

Comment:
Does the `ADIOS2_VERSION_MAJOR > 2` mean anything? there is no adios 3 yet right?

Reply:
You are correct, as per now `ADIOS2_VERSION_MAJOR` will always be 2. However, I recommend keeping it as this since new/future devs might be confused about this.